controllers/neutronapi_controller.go

```diff
@@ -922,6 +923,14 @@ func (r *NeutronAPIReconciler) reconcileNormal(ctx context.Context, instance *ne
 		return ctrl.Result{}, err
 	}
 
+	if instance.IsOVNEnabled() {
+		err = r.reconcileExternalOVNSecrets(ctx, helper, instance, &secretVars)
```

Comment:
maybe reconcile should be executed anyway - to clean up secrets if the driver was disabled?